src/main/java/org/folio/writer/impl/MarcRecordWriter.java

```diff
@@ -73,6 +82,7 @@ public String getResult() {
 
   @Override
   public List<VariableField> getFields() {
+    log.info("getFields:: ");
```

Comment:
This line doesn't contain any business-related information. Why do we need to know that namely this method starts execution? Add business related information or at least rename information string with sth like "Method NNN starting..."